src/Search.js

```diff
@@ -1,6 +1,7 @@
 import React from "react";
+import SearchButton from "./SearchButton.js";
 
-const Search = () => {
+const Search = props => {
```

Comment:
Where is this <Search /> component been used?

Reply:
on line 21 i think